launch/launch/launcher.py

```diff
@@ -27,14 +27,12 @@
 
 
 class _TaskException(Exception):
-
     def __init__(self, task_descriptor_index, exception):
         self.task_descriptor_index = task_descriptor_index
         self.exception = exception
 
 
 class DefaultLauncher(object):
-
```

Comment:
IIRC, it was a change in PEP257. They used to say to put this blank line in so it matched classes with docstrings which have a blank line between the class definition and the dosctring. But since then they changed that so that you don't put a blank line between the class definition and the docstring so this blank line is no longer required. I don't think it is explicitly forbidden, but personally I wouldn't have added it if writing the class from scratch:

https://hg.python.org/peps/rev/9b715d8246db

Maybe related: https://github.com/ament/ament_lint/pull/47

I don't think it matters if they're removed or not, so don't spend anymore time changing these.